src/components/org/PositionCard/components/TimelineInstance.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useMemo } from 'react';
+import styles from '../styles.less';
+import { PositionInstance } from '@equinor/fusion';
+import { useTooltipRef } from '@equinor/fusion-components';
+import classNames from 'classnames';
+
+type TimelineInstanceProps = {
+    instance: PositionInstance;
+    currentInstance: PositionInstance | null;
```

Comment:
What's the difference between `instance` and `currentInstance`?

Reply:
instance is the instance from the frominstances map(), currentInstance is the current

Reply:
Could the naming of these be changed to be less ambiguous?

Reply:
Adde jsDoc